src/IO/InterfileHeader.cxx

```diff
@@ -1346,6 +1350,12 @@ bool InterfilePDFSHeader::post_processing()
     }
   //cerr << data_info_ptr->parameter_info() << endl;
   
+  // If there is a non-zero bed position, set it
```

Comment:
I don't think we should do the `if`. I'm not sure what the default should be for `ProjDataInfo`. We can argue about this but I think it's fine to say "default is zero".
In any case, using 0 to check for an unset value seems rather tricky as it means it cannot be set to zero...